RecoEgamma/EgammaPhotonProducers/python/mustacheOOTPhotonCore_cff.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+from RecoEgamma.EgammaPhotonProducers.photonCore_cfi import *
+
+mustacheOOTPhotonCore = photonCore.clone()
+mustacheOOTPhotonCore.scHybridBarrelProducer = cms.InputTag("particleFlowSuperClusterOOTECAL:particleFlowSuperClusterOOTECALBarrel")
+mustacheOOTPhotonCore.scIslandEndcapProducer = cms.InputTag("particleFlowSuperClusterOOTECAL:particleFlowSuperClusterOOTECALEndcapWithPreshower")
+mustacheOOTPhotonCore.conversionProducer = cms.InputTag("conversions")
```

Comment:
is there a good reason to prefix "mustache" ?
is there a plan to add another type of OOT photons? (I thought no)